src/14_cal.py

```
@@ -21,4 +21,29 @@
 
 import sys
 import calendar
-from datetime import datetime
\ No newline at end of file
+from datetime import datetime
+
+user_input = input('Enter a month and/or year separated by commas: ').split(',')
+
```

you can use sys.argv to grab the user input as well. That way they don't have to format it in a particular way. That way you can just track the length of sys.argv that will determine what to return. 

If length is 1 -- that means the only argv provided was the filename...so return current month and current year

If length is 2 -- that means user didn't specify year...so return their month with current year

if length is 3 -- that means user supplied both month and year

if input is faulty -- you can return an error message in terminal.